canso-data-plane/canso-agent/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.10
+version: 0.1.11
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v0.0.6-python-3.8-slim"
+appVersion: "v0.0.7-python-3.13-slim"
```

Comment:
In the dev agent, the latest tag is v0.0.10 in the PR, and it should remain consistent.
Ref:- https://github.com/Yugen-ai/platform-dev-agent/pull/30/files

Reply:
@ashishYugen Is it the case that we forgot to bump the app version in this chart before and therefore it is `0.0.6` and not `0.0.9`?

Reply:
Yes, this will not affect the latest changes or version bumps, but it should be bumped as well.